src/chunkserver/datastore/chunkserver_datastore.h

```diff
@@ -173,6 +173,19 @@ class CSDataStore {
      */
     virtual CSErrorCode DeleteSnapshotChunkOrCorrectSn(
         ChunkID id, SequenceNum correctedSn);
+
+    // Deprecated, only use for unit & integration test
```

Comment:
Inconsistent comment format and context.

Reply:
done